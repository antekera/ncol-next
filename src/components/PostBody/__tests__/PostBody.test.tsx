import { render } from '@testing-library/react'
import { PostBody } from '..'

describe('PostBody', () => {
  test('should match snapshots', () => {
    const { container } = render(
      <PostBody
        firstParagraph='<p>Lorem ipsum</p>'
        secondParagraph='<p>Lorem ipsum</p>'
      />
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
