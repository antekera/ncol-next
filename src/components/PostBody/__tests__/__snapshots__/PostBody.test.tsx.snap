// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostBody should match snapshots 1`] = `
<div
  class="capital content capital-letter post-body mx-auto max-w-2xl"
>
  <p>
    Lorem ipsum
  </p>
</div>
`;
